### PR TITLE
[MRI Violations] Fix Time Run date filtering

### PR DESCRIPTION
**Pull request by davidblader**
https://redmine.cbrain.mcgill.ca/issues/13022
MRI Violations was restricting date search within the range of [`startYear - ageMax`, `endYear - ageMin`] which for CCNA would be [1916, 1970]
Would be ideal to be able to search for scans that occured after the invention of  MRI technology

### CODE REVIEW

**Comment by davidblader** [comment]
i'm thinking we should get rid of start and end date restrictions entirely but i'm not sure if they were put there for a specific reason or not

**Comment by ridz1208** [comment]
@MounaSafiHarab can you test please

**Comment by MounaSafiHarab** [comment]
@ridz1208 

will do. We need though to ensure the reason these dates were set as such. 
@davidblader changes make sense, but could be too restrictive for some projects with a very narrow search window?
we can also change the high end of the date search to be `endYear + ageMin`

**Comment by ridz1208** [comment]
After brief discussion with @LeighMac @driusan and @MounaSafiHarab the consensus is the following :+1: 
`$minYear     = $config->getSetting('startYear') - $config->getSetting('ageMax');`
`$maxYear = $config->getSetting('endYear');`

**Comment by MounaSafiHarab** [comment]
> After brief discussion with @LeighMac @driusan and @MounaSafiHarab the consensus is the following :
> $minYear = $config->getSetting('startYear') - $config->getSetting('ageMax');
> $maxYear = $config->getSetting('endYear');

This filter is filtering by the date the insertion was run (Time run), it has nothing to do with the scan date. I am not even sure why this would even be linked to the candidate age in any way.

**Comment by MounaSafiHarab** [comment]
@davidblader 
all good now, we can now search on CCNA between 1916 and 2020!

File: modules/mri_violations/php/NDB_Menu_Filter_Form_resolved_violations.class.inc [hunk 1]
```diff
@@ -328,9 +328,9 @@ class NDB_Menu_Filter_Form_Resolved_Violations extends NDB_Menu_Filter_Form
         // create user object
         $user        =&        User::singleton();
         $config      =&        NDB_Config::singleton();
-        $study       = $config->getSetting('study');
+        $study       = $config->getSetting('Study');
```
Comment (driusan) [inline modules/mri_violations/php/NDB_Menu_Filter_Form_resolved_violations.class.inc]: Why is this changed to upper case? It's lower case in the schema (and everywhere else in the codebase..)

Comment (MounaSafiHarab) [inline modules/mri_violations/php/NDB_Menu_Filter_Form_resolved_violations.class.inc]: @driusan 
It was not working for me with small `s` and worked with capital `S`. But we now know why, so @davidblader will issue a fix.

**Comment by MounaSafiHarab** [comment]
re-tested after the last commit and it is all good.
